pandas/core/dtypes/concat.py

```diff
@@ -38,6 +40,9 @@
     )
 
 
+_dtype_obj = np.dtype(object)
```

Comment:
why is this needed?

Reply:
i think its leftover from a previous version.  will remove next time i merge main